rpm/pacemaker.spec.in

```diff
@@ -177,6 +177,8 @@
 
 ## Different distros name certain packages differently
 ## (note: corosync libraries also differ, but all provide corosync-devel)
+%global hacluster_id_suse 90
+%global hacluster_id_other 189
```

Comment:
The only place we currently use hacluster_id is for creating the group and user.  What if we instead have two different pacemaker.sysusers in the source tree - one for suse and one for other - and we pick the correct one using the `Source1` and `Source2` trick below? I think we could then get away with not doing any of the cat/sed tricks.

Reply:
Yeah I thought through both variants. If going with 2 pacemaker.sysusers files would allow us to go with static files decision would be simple but they would have to be in the toplevel-src-directory which is ugly. So we would at least have to copy them over. Then I thought I could treat them a bit on that way. That was actually before I found that I have to create 2 versions to get a universal srpm and when I found that the cat/sed trickery was there already. And then it was keeping the distro specific stuff in the spec file vs. no cat/sed trickery - which would weigh heavier and if the 2nd was worth the effort to revert everything.